imports/plugins/included/wallet/client/settings/containers/walletSettingsFormContainer.jsx

```diff
@@ -0,0 +1,144 @@
+import React, {
+  Component
+} from "react";
+import PropTypes from "prop-types";
+import {
+  composeWithTracker
+} from "@reactioncommerce/reaction-components";
+import {
+  Meteor
+} from "meteor/meteor";
+import {
+  Packages
+} from "/lib/collections";
+import {
+  TranslationProvider
+} from "/imports/plugins/core/ui/client/providers";
+import {
+  Reaction,
+  i18next
+} from "/client/api";
+import {
+  WalletSettingsForm
+} from "../components";
+
+/**
+  *
+  * @class WalletSettingsFormContainer
+  * @extends {Component}
+*/
+class WalletSettingsFormContainer extends Component {
+  /**
+    * Creates an instance of WalletSettingsFormContainer.
+    * @param {any} props
+    * @memberof WalletSettingsFormContainer
+  */
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      apiKey: "278302390293"
```

Comment:
Is there no config file where we should be setting the apiKey. This kind of data should not be hard coded. Research on how to use `.env` to create environment variables and use it to store this apiKey